platform/platform-app/src/main/resources/features.xml

```diff
@@ -1046,7 +1046,7 @@
         <bundle>mvn:ddf.platform/platform-parser-xml/${project.version}</bundle>
     </feature>
 
-    <feature name="platform-filter-delegate" install="manual" version="${project.version}">
+    <feature name="platform-filter-delegate" install="auto" version="${project.version}">
```

Comment:
I was talking with @emmberk about this and we don't think this will make the feature start any sooner. The auto value is only used when a kar file is dropped in or by our own application installer but this won't affect the boot feature startup time/order.

Reply:
Then I can't figure out why I haven't been able to reproduce the issue with this change but I can as soon as I revert it 😭

Reply:
Following up for our conversation, after double checking with the Documentation (see section `4.10.12. Deployer` in https://karaf.apache.org/manual/latest/), I wouldn't think that `install="auto"` would make a difference here because we're not hot deploying the features xml file. I'd like to see other solutions explored, like testing out different configurations of prereqs and boot features. On the other hand, `install="auto"` might be working for some reason that I don't understand, and I don't think it would hurt anything to merge this in (with a code comment) if it seems to fix the issue.